src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -303,24 +310,23 @@ interface StatusCellProps extends OpenSpotAlertPopoverProps {
 const StatusCell = withStyles(styles)((props: StatusCellProps) => {
     const { term, sectionCode, courseTitle, courseNumber, status, classes } = props;
 
-    if (term === getDefaultTerm().shortName && (status === 'NewOnly' || status === 'FULL')) {
-        return (
-            <NoPaddingTableCell className={`${classes[status.toLowerCase()]} ${classes.cell}`}>
-                <OpenSpotAlertPopover
-                    courseTitle={courseTitle}
-                    courseNumber={courseNumber}
-                    status={status}
-                    sectionCode={sectionCode}
-                />
-            </NoPaddingTableCell>
-        );
-    } else {
+    // TODO: Implement course notification when PeterPortal has the functionality, according to #473
```

Comment:
Good change but can you add it to the PR description so it's documented when we commented this out? If I wanted to nitpick super hard I'd say to remove this change and open a new PR for it but honestly it's small enough that I'll let it slip.

Reply:
I think I was on the wrong branch when I made that change. I'll put that in the PR desc.